nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/components/explorer/dialogs/about/nf-registry-explorer-about.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Inject } from '@angular/core';
+import { MatDialog, MAT_DIALOG_DATA } from '@angular/material/dialog'
+
+export interface DialogData {
+    registryAboutVersion: string
+}
+
+@Component({
+    selector: 'nifi-registry-explorer-about-dialog',
+    templateUrl: './nf-registry-explorer-about.html'
+})
+export class NfRegistryExplorerAbout {
+    /**
+     * NfRegistryExplorerAbout constructor.
+     *
+     * @param DialogData - The data to be injected into the dialog
```

Comment:
Please update the doc.
```suggestion
     * @param data - The data to be injected into the dialog
```